R/loadComp.R

```diff
@@ -397,7 +397,7 @@ predictSolute.loadComp <- function(
 #'   points are needed to make this estimation with precision.
 #' @export
 #' @family estimateMSE
-estimateMSE.loadComp <- function(load.model, n.iter=100, method="parametric", rho) {
+estimateMSE.loadComp <- function(load.model,..., n.iter=100, method="parametric", rho) {
```

Comment:
glad you were able to eliminate the error, but this strikes me as an odd UI. could you experiment with placing the dots at the end for this and the other estimateMSE definitions? My guess is you'll also have to move all the inconsistent arguments to the end; that's OK with me. Please do a search of estimateMSE through the whole package to fix any places where the function is called and the change in argument orders means that the arguments passed in the function call should also be changed.